extensions/amp-bind/0.1/validator-amp-bind.protoascii

```diff
@@ -46,39 +46,20 @@ tags: {  # <amp-state> (json)
   }
   spec_url: "https://www.ampproject.org/docs/reference/components/amp-bind"
 }
-tags: {  # <amp-state> with json child
```

Comment:
@honeybadgerdontcare I thought about that as well, but then `<amp-state>` elements with both `src` and a json child will validate against the wrong rule (the src only rule) which doesn't require the `<amp-state> json` rule.

This is presuming we switch the order so that the src only rule comes first.